src/lib/seam/components/AccessCodeTable/AccessCodeRow.tsx

```diff
@@ -63,9 +77,14 @@ export function AccessCodeRow({
         <AccessCodeMenu
           accessCode={accessCode}
           onEdit={onEdit}
+          onDeleteSuccess={onDeleteSuccess}
           onViewDetails={onClick}
-          disableDeleteAccessCode={disableDeleteAccessCode}
-          disableEditAccessCode={disableEditAccessCode}
+          disableDeleteAccessCode={
+            isAccessCodeBeingRemoved || disableDeleteAccessCode
+          }
+          disableEditAccessCode={
+            isAccessCodeBeingRemoved || disableEditAccessCode
+          }
```

Comment:
Will this hide the button?

Reply:
Move the logic within the component as mentioned during our call